scripts/upgrade

```diff
@@ -96,40 +97,66 @@ tempdir="$(mktemp -d)"
 ynh_setup_source --dest_dir=$tempdir --source_id=$architecture
 ynh_exec_warn_less dpkg -i $tempdir/duniter-server-v1.8.*-linux-*.deb
 
+#=================================================
+# MOVE DUNITER DIRECTORY
+#=================================================
+datadir=/home/yunohost.app/$app
+if [[ ! -d "${datadir}" ]]; then
+	ynh_script_progression --message="Moving data directory…"
+	mv /root/.config/duniter $datadir
+	chown duniter: $datadir
+fi
```

Comment:
Maybe you should manage the `chmod` and `chown` on datadir after creating it

Reply:
Done.